docs/crio.conf.5.md

```diff
@@ -112,6 +112,9 @@ The `crio.runtime` table contains settings pertaining to the OCI runtime used an
   Additional environment variables to set for all the containers. These are overridden if set in the container image spec or in
 the container runtime configuration.
 
+**pull-in-a-cgroup**=false
+  If true, image pulls will happen in a new cgroup. It is currently supported only with the systemd cgroup manager.
```

Comment:
I feel like this should be in the `[crio.image]` table...
also, can we rename to `separate-pull-cgroup` or something like that?